chart/templates/dashboards/openstack-dashboard.yaml

```diff
@@ -0,0 +1,6876 @@
+#
+# Copyright 2020 The dNation Kubernetes Monitoring Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+{{- if and .Values.grafanaDashboards.enable .Values.openstackMonitoring.enabled }}
+{{- $labelKey := first (keys .Values.grafanaDashboards.labelGrafana) }}
+{{- $labelValue := first (values .Values.grafanaDashboards.labelGrafana) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ include "k8s-monitoring.namespace" . }}
+  name: {{ include "k8s-monitoring.fullname" . }}-openstack-exporter-http-dashboard
```

Comment:
Please adjust the name of the configmap. It has nothing to do with `http`, so why is it there?